pkgs/top-level/all-packages.nix

```diff
@@ -6855,6 +6855,17 @@ with pkgs;
 
     mozc = callPackage ../tools/inputmethods/ibus-engines/ibus-mozc { };
 
+    mozc-ut = callPackage ../tools/inputmethods/ibus-engines/ibus-mozc { dictionaries = [
```

Comment:
Why not just make these the default value of the `dictionaries` variable in the deriv, like you've done in `merge-ut-dictionaries`?

Reply:
I'm unsure if it is a good design, but I wanted to show that dictionaries should be explicitly added and keep them out of the mozc derivation.
Further, I think it may be better if merge-ut-dictionaries were a function that takes some dictionaries and outputs a file. I wasn't quite sure where to put something like that in the by-name directory. Is there a place to put a file like that?